src/Microsoft.Azure.CosmosEventSourcing/Exceptions/InvalidEventItemPartitionKeyAttributeCombinationException.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.CosmosEventSourcing.Attributes;
+
+namespace Microsoft.Azure.CosmosEventSourcing.Exceptions;
+
+/// <summary>
+/// Exception which is thrown the PersistAsync is used with the aggregate overload and the aggregate provides multiple InvalidEventItemPartitionKeyAttributeCombinationException
+/// </summary>
+public class InvalidEventItemPartitionKeyAttributeCombinationException : Exception
+{
+    /// <summary>
+    /// Constructor
```

Comment:
```suggestion
    /// Ceates a new <see cref="InvalidEventItemPartitionKeyAttributeCombinationException" />
```
Same for the other 3

Reply:
Updated